dockerfile-configs/common-components.yaml

```diff
@@ -70,7 +70,10 @@
     to: /nerdctl.tar.gz
     command: |
       tar Cxzvvf /usr/local/bin nerdctl.tar.gz &&\
-      rm -f nerdctl.tar.gz
+      rm -f nerdctl.tar.gz &&\
+      mkdir /etc/nerdctl &&\
+      echo address = \"unix:///host/run/containerd/containerd.sock\" >> /etc/nerdctl/nerdctl.toml &&\
+      echo namespace = \"k8s.io\" >> /etc/nerdctl/nerdctl.toml
     info: nerdctl is a Docker-compatible CLI for containerd. If the root directory of the host is mounted you can use the files under /hostroot/run/ to connect to the container runtime of the host e.g. 'nerdctl --address unix:///hostroot/run/containerd/contaierd.sock ps'
```

Comment:
you were also setting the namespace which we could still set I guess.
However we should only set the address if we align the path to either be `/hostroot` or `/host`

Reply:
Sure, but if we are setting then can we align on the address as well to be either /hostroot or /host. I'm definitely not the right person to take this call. So i'll go with your judgement here.

Reply:
@plkokanov @dguendisch what would be your preference? 
- `/hostroot`
  - https://github.com/gardener/terminal-controller-manager/blob/master/controllers/terminal_controller.go#L1206
  - https://github.com/gardener/dashboard/blob/a5f8a9f273bffc652a15457ba926fc8fa1696f58/frontend/src/components/TerminalSettings.vue#L45
- or `/host`
  - https://github.com/gardener/ops-toolbelt/blob/master/hacks/ops-pod#L177

Reply:
Personal preference, +1 for `/host` (and +1 for setting it in the `nerdctl.toml` so that it works out of the box :) )

Reply:
+1 for `/host` as well

Reply:
thanks for your input. I will adapt the tcm and dashboard accordingly. @ashwani2k , could you change the info to be /host instead of /hostroot https://github.com/gardener/ops-toolbelt/blob/706db5ccb088a16dff124c551a65ff5f8cbde863/dockerfile-configs/common-components.yaml#L77?